src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -13,6 +13,8 @@
 
 use PHPUnit\Framework\MockObject\Matcher\StatelessInvocation;
 
+@trigger_error(sprintf('The "%s" class is deprecated since Symfony 4.4, use "Symfony\Component\ErrorHandler\DebugClassLoader" instead.', DebugClassLoader::class), E_USER_DEPRECATED);
```

Comment:
please also use `::class` for the alternative class as well for discoverability, autocomplete etc. you can use the FCQN::class to avoid the name conflict.